doc/templates/index.html

```diff
@@ -169,6 +169,8 @@ <h4 class="sk-landing-call-header">News</h4>
         <li><strong>On-going development:</strong>
         <a href="https://scikit-learn.org/dev/whats_new.html"><strong>What's new</strong> (Changelog)</a>
         </li>
+	<li><strong>September 2023.</strong> scikit-learn 1.3.1 is available for download (<a href="whats_new/v1.3.html#version-1-3-1">Changelog</a>).
+        </li>
```

Comment:
with every item we add, we can probably remove an item from the end of the list.

Reply:
let's decide on a release window and write it in the maintainers guide. What do you think about keeping all entries since the previous major release ? for 1.3.2 it means we keep everything since 1.2.0. In general it means for 1.X.Y we keep everything since 1.(X-1).0.

Reply:
that sounds good, so we don't have to remove anything now.